### PR TITLE
MINOR: Update dependencies for 1.0.0 release

### PR DESCRIPTION
**Pull request by andrasbeni**
Updates dependencies to their latest versions.

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -50,10 +50,10 @@ versions += [
   apacheda: "1.0.0",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  bcpkix: "1.57",
+  bcpkix: "1.58",
   easymock: "3.4",
   jackson: "2.8.5",
-  jetty: "9.2.15.v20160210",
+  jetty: "9.4.6.v20170531",
```
Comment (ijuma) [inline gradle/dependencies.gradle]: This requires Java 8, so we can't do it yet.

File: gradle/dependencies.gradle [hunk 2]
```diff
@@ -50,10 +50,10 @@ versions += [
   apacheda: "1.0.0",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  bcpkix: "1.57",
+  bcpkix: "1.58",
   easymock: "3.4",
   jackson: "2.8.5",
-  jetty: "9.2.15.v20160210",
+  jetty: "9.2.22.v20170606",
```
Comment (ijuma) [inline gradle/dependencies.gradle:56]: @xvrl any concerns with upgrading to this Jetty version? You mentioned previously some issues with certain versions. cc @ewencp

**Comment by ijuma** [comment]
Is it intentional that `org.glassfish.jersey.containers:jersey-container-servlet [2.25.1 -> 2.26]` was not included?

File: gradle/dependencies.gradle (hunk 2 above)
Comment (xvrl) [reply gradle/dependencies.gradle:56]: @ijuma My info is a bit dated. I don't have much info on the most recent versions of Jetty 9.2.x. The projects I know of have moved on to Java 8 and Jetty 9.3.x.

Comment (xvrl) [reply gradle/dependencies.gradle:56]: A lot of bugs have probably been fixed so I would upgrade and rollback later if necessary.

**Comment by andrasbeni** [comment]
It was not released when I first made the changes. I could include it,
however.

**Comment by ijuma** [comment]
Yes, let's do it.

File: gradle/dependencies.gradle (hunk 2 above)
Comment (ijuma) [reply gradle/dependencies.gradle:56]: Thanks @xvrl

**Comment by andrasbeni** [comment]
Unfortunately https://jersey.github.io/documentation/latest/modules-and-dependencies.html says "Since Jersey 2.26, all modules are build using Java SE 8 and there is no support for running it on older Java SE distributions."
It seems this upgrade has to wait.

**Comment by ijuma** [comment]
I thought that may have been the case.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. Merging to trunk.

**Pull request closed without merging**